src/main/java/org/openrewrite/kotlin/internal/KotlinTreeParserVisitor.java

```diff
@@ -2339,43 +2361,50 @@ public J visitDestructuringDeclaration(KtDestructuringDeclaration multiDeclarati
     @Override
     public J visitDotQualifiedExpression(KtDotQualifiedExpression expression, ExecutionContext data) {
         assert expression.getSelectorExpression() != null;
+        Space prefix = deepPrefix(expression);
```

Comment:
@kunli2 There were a few missing calls to deepPrefix, which I added, and it seems fine.
But I don't know when different prefix methods need to be used.

Is it okay to call deepPrefix on each returned element?